src/client/components/robot_selector/style.css

```diff
@@ -0,0 +1,62 @@
+.button {
+  align-items: center;
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  font-size: 0.7rem;
+  height: 60px;
+  justify-content: center;
+  outline: 0;
+  padding: 0 1em;
+  transition: background-color 0.3s ease;
+}
+
+.button:hover {
+  background-color: #f9f9f9;
+}
+
+.button > svg {
+  fill: #1197d3;
+  height: 2.85em;
+  margin: 0 auto;
+  padding: 0.25em;
+  transition: fill 0.25s ease-out;
+  width: auto;
+}
+
+.button:hover > svg {
+  fill: #0d76a6;
+}
+
+.robots {
+  background-color: white;
+  border-bottom-left-radius: 2px;
+  border-bottom-right-radius: 2px;
+  box-shadow:
+    0px 0px 3px 0px rgba(0, 0, 0, 0.15),
+    0px 3px 8px 0px rgba(0, 0, 0, 0.15);
+  width: 100%;
+}
+
+.robot {
+  align-items: center;
+  display: flex;
+  padding: 1em;
+}
+
+.robot:hover {
+  background-color: #eee;
+  cursor: pointer;
+}
+
+.robot:last-child:hover {
```

Comment:
If you add `overflow: hidden` to the `. robots` class you can avoid/delete this. A bit less coupling between the classes.